sources/HUBContainerView.h

```diff
@@ -26,6 +26,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// View that acts as a container view for a Hub Framework view controller
 @interface HUBContainerView : UIView
 
+/// Collection view contained by the container
```

Comment:
Suggestion: Add a note that setting this property also adds the collection view to the container view as a subview?